### DISCUSSION TITLE
Retrieval mining newbie

### DISCUSSION THREAD

**Discussion by olongapsi** [filecoin-project/community#178] [Mining]
Hello! I am new to retrieval mining. I understand that storage mining has already started, but retrieval mining has not yet started. Is that correct? Just wondering when retrieval mining will start.

**Comment by rjan90** [comment]
Hey! That’s correct. A retrieval-miner is currently in development: https://youtu.be/QuE_NotFCXs !

As for how long until it is ready, I’m not sure, but it will probably take some time

**Comment by olongapsi** [comment]
Thank for the link, that’s really useful info. 
Are storage deals being made now? And are clients already paying fees for storage?

**Reply by rjan90** [reply]
Yes, and yes 👍 You can see deals being made here: https://filfox.info/en/deal. Retrievals are also working for storage-miners, it is the standalone retrieval-miner that is currently in development

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
to be clear on this:

retrieval mining will not get block rewards but "survive" on transfer fees

**Reply by akjinju** [reply]
Who will pay the transfer fees? 
Is there more info somewhere about how the retrieval mining payment system is intended to work?

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
the retrieval client will pay - the one who pulls the data

https://spec.filecoin.io/#section-systems.filecoin_markets.retrieval_market

**Reply by akjinju** [reply]
Thanks
